standard/abstract_tests/core/ATS_test_links.adoc

```diff
@@ -24,7 +24,7 @@ For each link object, check that the `+href+` property contains a valid protocol
 
 [.component,class=step]
 --
-Check that the `+links+` array property contains one link object with a ``rel`` property with one of the values ``canonical``, ``http://def.wmo.int/def/rel/wnm/-/update``, ``http://def.wmo.int/def/rel/wnm/-/deletion`` as well as an ``href`` property.
+Check that the `+links+` array property contains one link object with a ``rel`` property with one of the values ``canonical``, ``https://codes.wmo.int/wis/link-type/update``, ``https://codes.wmo.int/wis/link-type/deletion`` as well as an ``href`` property.
```

Comment:
As discussed at the meeting last week, including a full URL in the "rel" property is inconsistent with the other types of "rel" values, such as "canonical" and "item". I think we should change this to just "update" and "deletion"  for consistency.

Reply:
Updated.  We need to revisit this following INFCOM-3.  I've opened #47 for good measure (perhaps moving to `properties.operation` is safer in this regard).